src/main/webapp/style.css

```diff
@@ -6,3 +6,23 @@
   color: #858796;
   font-size: 15px;
 }
+
+.pagination {
+  display: inline-block;
```

Comment:
why do you need to override all of this? by default, I think `.pagination` applies a `display: flex`, and you wouldn't need to `float: left` below. I think the key is that you need to add the `page-item` class to each item. (Also, probably worth it to swap the outer element to a `<ul>`

https://getbootstrap.com/docs/4.0/components/pagination/